assets/base.css

```diff
@@ -1509,21 +1542,44 @@ details[open] > .share-button__fallback {
   -webkit-appearance: none;
   appearance: none;
   background-color: rgb(var(--color-background));
-  border: 0.1rem solid transparent;
-  border-radius: var(--inputs-radius);
   color: rgb(var(--color-foreground));
   font-size: 1.6rem;
   width: 100%;
-  box-shadow: 0 0 0 var(--inputs-border-width) rgba(var(--color-foreground), var(--inputs-border-opacity));
-  height: 4.5rem;
   box-sizing: border-box;
   transition: box-shadow var(--duration-short) ease;
-  filter: drop-shadow(
-    var(--inputs-shadow-horizontal-offset)
-    var(--inputs-shadow-vertical-offset)
-    var(--inputs-shadow-blur-radius)
-    rgba(var(--color-shadow), var(--inputs-shadow-opacity))
-  );
+  border-radius: var(--inputs-radius);
+  min-height: calc(var(--inputs-border-width) * 2);
```

Comment:
I still think the input height isn't quite right. Currently on main, inputs and buttons are 45px tall by default with no additional border. With these changes we're sitting at 54px. This is likely why we needed to adjust the vertical alignment of the search icon. We may just need to reduce the 1.5rem padding a bit. I'd expect the min-height here to be calculated to 45px.
<img src="https://screenshot.click/11-35-jymf0-j147b.png" width="350"/>

Reply:
The height is bigger because now we possibly have thicker borders. If I keep the height always the same, because the borders grow "inwards", you end up with less space for the content which isnt desired.

<img width="878" alt="Screen Shot 2022-01-11 at 2 11 20 PM" src="https://user-images.githubusercontent.com/19962891/149006378-d28f0df4-92d6-45a2-b541-f395f8a51b51.png">

Thinner borders just means we have less margin:
<img width="878" alt="Screen Shot 2022-01-11 at 2 11 20 PM" src="https://user-images.githubusercontent.com/19962891/149006481-b00db10f-32b5-43c8-adde-d8bb5b4b6e53.png">

But youll notice the height and padding remain the same

Reply:
Yeah we can't reduce the inner height based on the border, but with only 1px of border the inputs height is still 54px now, as opposed to 45. Reducing this "base" height back down to where it was may also give [this issue](https://camo.githubusercontent.com/e673e7770868e3fbce2be1a2cf30b18a2fc291218c5f7fa51a2f179d8d23fc2e/68747470733a2f2f73637265656e73686f742e636c69636b2f32322d30312d6b757a31692d79717878332e706e67) more room

Reply:
This is what I mean 
<img src="https://screenshot.click/11-27-xj60l-7rwm6.png" width="350"/>

Reply:
Good catch, Ken! This is now fixed